pkg/processor/eventprocessor/statistics.go

```diff
@@ -32,29 +32,23 @@ type AllocatorStatistics struct {
 	AllocationObjectsAvailablePercentage  uint64
 }
 
-func (s *AllocatorStatistics) DiffFrom(prev *AllocatorStatistics) AllocatorStatistics {
+type safeAllocatorStatistics struct {
```

Comment:
Why not replace `AllocatorStatistics` completely? where is it still useful?

Reply:
@TomerShor we don't need statistics object to be safe in TriggerGatherer

Reply:
TriggerGatherer has its own `Statistics` object, but I don't think we need keep both `safeAllocatorStatistics` and `AllocatorStatistics` objects

Reply:
@TomerShor are you suggesting replacing it with atomic version even though we don't need it to be atomic there?